std/security/ecc-signer.go

```diff
@@ -9,38 +9,17 @@ import (
 	enc "github.com/named-data/ndnd/std/encoding"
 	basic_engine "github.com/named-data/ndnd/std/engine/basic"
 	"github.com/named-data/ndnd/std/ndn"
-	"github.com/named-data/ndnd/std/utils"
 )
 
 // eccSigner is a signer that uses ECC key to sign packets.
 type eccSigner struct {
```

Comment:
nitpicking on the naming: "ECC" is a very broad category of cryptography but this code is specifically for ECDSA. Saying "ecc signing" or "ecc key" is fairly meaningless (what algorithm?). Moreover, EdDSA also uses elliptic curves (twisted Edwards curves), so this naming could be confusing. The comment above is also meaningless for similar reasons.